app/src/main/java/io/homeassistant/companion/android/widgets/StaticWidgetConfigureActivity.kt

```diff
@@ -163,6 +158,16 @@ class StaticWidgetConfigureActivity : Activity() {
                 label.text.toString()
             )
 
+            intent.putExtra(
+                StaticWidget.EXTRA_TEXT_SIZE,
+                textSize.text.toString()
```

Comment:
Shouldn't this be a number? `Float` or `Int`?

Reply:
It is a number and I changed it to float in DB, however we are using a text field for input. Yes, you can only input numbers as specified in xml but as it is text field, it has to be a String. It is later converted to float when saving to DB.